scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+export SKIP_USER_LOGIN_TESTS="true"
+export GINKGO_TEST_ARGS="--noColor"
+
+shout "Getting oc binary"
+if [[ $BASE_OS == "linux"  ]]; then
+    set +x
+    curl --connect-timeout 150 --max-time 2048 -k ${OCP4X_DOWNLOAD_URL}/${ARCH}/${BASE_OS}/oc.tar -o ./oc.tar
+    set -x
+    tar -C $GOBIN -xvf ./oc.tar && rm -rf ./oc.tar
+else
+    set +x
+    curl --connect-timeout 150 --max-time 2048 -k ${OCP4X_DOWNLOAD_URL}/${ARCH}/${BASE_OS}/oc.zip -o ./oc.zip
+    set -x
+    if [[ $BASE_OS == "windows" ]]; then
+        GOBIN="$(cygpath -pw $GOBIN)"
+        CURRDIR="$(cygpath -pw $WORKDIR)"
+        powershell -Command "Expand-Archive -Path $CURRDIR\oc.zip  -DestinationPath $GOBIN"
+        chmod +x $GOBIN/*
+    fi
+    if [[ $BASE_OS == "mac" ]]; then
+        unzip ./oc.zip -d $GOBIN && rm -rf ./oc.zip && chmod +x $GOBIN/oc
+        PATH="$PATH:/usr/local/bin:/usr/local/go/bin"
+    fi
+fi
+
+PATH=$PATH:$GOBIN
+
+#-----------------------------------------------------------------------------
+
+shout "Testing"
+
+# Run unit tests
+GOFLAGS='-mod=vendor' make test
```

Comment:
Shouldn't we `shout` that we're doing unit testing?

Reply:
Updated